src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,63 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_VALUE = 10;
+    private int createdStorages;
+    private Object[] keyArrayList = new Object[MAX_STORAGE_VALUE];
+    private Object[] valueArrayList = new Object[MAX_STORAGE_VALUE];
+
+    private K key;
+    private V value;
+
+    private boolean contains(Object[] array, Object o) {
+        for (int i = 0; i < createdStorages; i++) {
+            if (Objects.equals(array[i], o)) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
do not use such naming(o), how to guess what it stores?
```suggestion
    private boolean contains(Object key) {
        for (int i = 0; i < size; i++) {
            if (Objects.equals(keys[i], key)) {
                return true;
            }
        }
        return false;
    }
```